firestore/incidents.py

```diff
@@ -114,11 +211,21 @@ def deleteIncident(incident_id):
         return True
     return False
 
-
-def getIncidents(start: datetime, end: datetime, state="", type="", self_report_status="", start_row="", page_size="", skip_cache=False):
+def getIncidents(start: datetime, end: datetime, state="", type="", self_report_status="", cursor=None, direction="forward", page_size=10, skip_cache=False):
     if skip_cache:
         INCIDENT_CACHE.clear()
-    return queryIncidents(start, end, state, type, self_report_status, start_row, page_size)
+    
+    # Convert cursor if it's a string
+    if cursor and isinstance(cursor, str) and cursor != "":
+        cursor = {'id': cursor}
+    
+    result = queryIncidents(start, end, state, type, self_report_status, page_size, cursor, direction)
+    
+    # If we got an error response, return it directly
+    if isinstance(result, dict) and "error" in result:
+        return result
+        
+    return result
```

Comment:
the if condition does not impact the result, we will return `result` regardless. so simply return result as the previous code does should be sufficient.